contracts/SSVNetwork.sol

```diff
@@ -123,7 +123,7 @@ contract SSVNetwork is UUPSUpgradeable, OwnableUpgradeable, ISSVNetwork {
         bytes calldata publicKey,
         uint256 fee
     ) external override returns (uint64 id) {
-        if (fee < MINIMAL_OPERATOR_FEE) {
+        if (fee != 0 && fee < MINIMAL_OPERATOR_FEE) {
```

Comment:
@mtabasco lets add a comment stating that 0 fee can't be changed and usually should be used with private operators